### PR TITLE
Fix typo in `readme.md`

### PR DESCRIPTION
**Pull request by trebeljahr**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -73,8 +73,8 @@ Now, running `node example` yields:
 
 ### `rehype().use(sanitize[, schema])`
 
-Remove potentially dangerous things from HTML, or more correct: keep only safe
-the things in a document.
+Remove potentially dangerous things from HTML, or more correct: keep only the safe 
+things in a document.
```
Comment (wooorm) [inline readme.md]:
```suggestion
Remove potentially dangerous things from HTML, or more correct: keep only the
safe things in a document.
```

The line became a bit too long. Thanks!!

**Comment by wooorm** [comment]
@trebeljahr Ping!

**Comment by trebeljahr** [comment]
@wooorm Why did you ping me? Is something wrong? Do I have to do something? ^^ Sry but I am a bit clueless regarding this whole Github open-source workflow stuff. I just wanted to fix that typo haha.

**Comment by wooorm** [comment]
@trebeljahr No problem! Yes, I pinged you because something was wrong: the tests are failing, see my suggestion on how to fix it here: https://github.com/rehypejs/rehype-sanitize/pull/5#discussion_r436373230

**Comment by trebeljahr** [comment]
Ah ok. I thought you had already applied those changes you suggested since they only included a line break and things were merged and done haha ^^

**Comment by trebeljahr** [comment]
@wooorm Ok. So anything else I have to do? Tests seem to pass now ^^

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
This is perfect, thanks!

**Pull request merged**